src/core/api/systeminfo.go

```diff
@@ -120,7 +122,8 @@ func (sia *SystemInfoAPI) validate() {
 func (sia *SystemInfoAPI) GetVolumeInfo() {
 	sia.validate()
 
-	capacity, err := config.AdminserverClient.Capacity()
+	systeminfo.Init()
```

Comment:
Please remove capacity related code from adminserver to core. One problem is you are relying env (e.g. `IMAGE_STORE_PATH`) of adminserver.

Reply:
The IMAGE_STORE_PATH is not set in adminserver, it is not set in core api,  uses default value /data.